src/core/deltafilewrapper.h

```diff
@@ -364,6 +384,23 @@ class DeltaFileWrapper : public QObject
      */
     QJsonValue attributeToJsonValue( const QVariant &value );
 
+
+    /**
+     * Append generated \a delta.
+     */
+    void appendDelta( const QJsonObject &delta );
+
+
+    /**
+     * Merge the generated \a delta into stored deltas.
+     */
+    void mergeDelta( const QJsonObject &delta );
+
+    void mergeCreateDelta( const QJsonObject &delta );
+    void mergeDeleteDelta( const QJsonObject &delta );
+    void mergePatchDelta( const QJsonObject &delta );
```

Comment:
```suggestion

    /**
     * Merge the generated create \a delta into stored deltas. Should only be called from `mergeDelta` method.
     */
    void mergeCreateDelta( const QJsonObject &delta );


    /**
     * Merge the generated delete \a delta into stored deltas. Should only be called from `mergeDelta` method.
     */
    void mergeDeleteDelta( const QJsonObject &delta );


    /**
     * Merge the generated patch \a delta into stored deltas. Should only be called from `mergeDelta` method.
     */
    void mergePatchDelta( const QJsonObject &delta );
```

Just to make everything documented in this class :)